config/src/cluster_config.rs

```diff
@@ -70,27 +65,30 @@ impl ClusterConfig {
         if let Ok(name) = std::env::var("CNOSDB_CLUSTER_NAME") {
             self.name = name;
         }
-        if let Ok(meta) = std::env::var("CNOSDB_CLUSTER_META") {
-            self.meta_service_addr = meta;
-        }
-        if let Ok(id) = std::env::var("CNOSDB_NODE_ID") {
-            self.node_id = id.parse::<u64>().unwrap();
+        if let Ok(meta_list) = std::env::var("CNOSDB_CLUSTER_META") {
+            let tmp: Vec<&str> = meta_list.split(';').collect();
```

Comment:
`Split` is an iterator so we can skip that allocate for variable `tmp`.

#### Example

```rust
let a = "hello;world";
let mut b = vec![];
for c in a.split(';') {
    b.push(c.to_string());
}
```

Reply:
thanks.